### PR TITLE
Notes from board meeting April 23

### PR DESCRIPTION
**Pull request by martinmelin**
ping @anah and @strokirk

### CODE REVIEW

File: styrelsemote/styrelsemote-2015-04-23.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+If you are reading email in the info@pycon.se inbox, please archive email that has been fully handled, and star any email that needs more attention.
+
+Only 5 people had registered as Staff in Eventbrite - we need a complete list of people attending by next week to confirm food etc.
+
+- **everyone** attending the conference should register in Eventbrite
+
+### Next meeting
+
+Tuesday April 26, 20:00 Swedish time
```
Comment (nailor) [inline styrelsemote/styrelsemote-2015-04-23.md]: 28th?

Comment (martinmelin) [inline styrelsemote/styrelsemote-2015-04-23.md]: Oops, yes. Just making sure you're paying attention ;)

**Comment by strokirk** [comment]
+1!

**Comment by anah** [comment]
:+1:

**Pull request merged**